crates/provider-blobstore-azure/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "wasmcloud-provider-blobstore-azure"
+version = "0.1.0"
+description = """
+Blobstore for wasmCloud, leveraging Azure Blobstore. This package provides a capability provider that satisfies the 'wasi:blobstore' world.
+"""
+
+authors.workspace = true
+categories.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = { workspace = true }
+async-nats = { workspace = true }
+async-trait = { workspace = true }
+tokio = { workspace = true }
+path-clean = { workspace = true }
+serde = { workspace = true }
+tokio-stream = { workspace = true, features = ["fs"] }
+tokio-util = { workspace = true, features = ["io"] }
+tracing = { workspace = true }
+wasmcloud-provider-sdk = { workspace = true, features = ["otel"] }
+wrpc-interface-blobstore = { workspace = true }
+wrpc-transport = { workspace = true }
+base64 = { workspace = true }
+bytes = { workspace = true }
+serde_json = { workspace = true }
+simple_env_load = { workspace = true }
+time = { workspace = true }
+futures = { workspace = true }
```

Comment:
nit; could you sort these lexicographically? (e.g.:
```
vim --headless -c ':17,39sort i' -c ':wq' crates/provider-blobstore-azure/Cargo.toml
```
this will move your comment as well
)